docusaurus.config.js

```diff
@@ -1,306 +1,307 @@
 const createRedirects = require("./createRedirects");
 
-const config = {
-  title: "Mia-Platform Documentation",
-  tagline: "Learn how Mia-Platform can help you to develop your business",
-  url: "https://docs.mia-platform.eu",
-  baseUrl: "/",
-  onBrokenLinks: "throw",
-  onBrokenMarkdownLinks: 'throw',
-  favicon: "img/favicon.ico",
-  organizationName: "Mia-Platform", // Usually your GitHub org/user name.
-  projectName: "Mia-Platform", // Usually your repo name.
-  themeConfig: {
-    prism: {
-      additionalLanguages: ['rego','java', 'csharp'],
-    },
-    image: "img/documentation-link-preview.png",
-    algolia: {
-      placeholder: "Search for terms, features and more...",
-      apiKey: "602a752c6342891e2488bea38b0d9292",
-      appId: "58NJDUVYVW",
-      indexName: "mia-platform-docs",
-      contextualSearch: true,
-    },
-    colorMode: {
-      // "light" | "dark"
-      defaultMode: "light",
-    },
-    navbar: {
-      hideOnScroll: false,
-      title: "Mia-Platform Docs",
-      logo: {
-        alt: "Mia_Platform logo",
-        src: "img/logo.png"
+async function createConfig() {
+  const mdxMermaid = await import('mdx-mermaid')
```

Comment:
why the use of `import` rather than using require (as done at line 1); can we switch to imports all together?